catalog/app/containers/Bucket/PackageCopyDialog.js

```diff
@@ -84,11 +84,21 @@ function DialogTitle({ bucket }) {
 }
 
 const useStyles = M.makeStyles((t) => ({
+  form: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+    overflowY: 'auto',
+  },
   meta: {
+    display: 'flex',
+    flexDirection: 'column',
     marginTop: t.spacing(3),
+    overflowY: 'auto',
   },
 }))
 
+// <M.DialogContent /> doesn't have className prop
```

Comment:
but it has `classes` prop, so you can pass a class using `classes={{ root: className }}`

Reply:
I forgot it, thanks